odoo/api.py

```diff
@@ -288,6 +288,15 @@ def split_context(method, args, kwargs):
     return kwargs.pop('context', None), args, kwargs
 
 
+def autovacuum(method):
+    """
+    Decorate a model-style method so it is registered as part of the
+    daily housekeeping done by :meth:`ir.autovacuum`.
```

Comment:
```suggestion
    Decorate a method so that it is called by the daily vacuum cron job (model
    ``ir.autovacuum``).  This is typically used for garbage-collection-like
    tasks that do not deserve a specific cron job.
```